pages/Product.vue

```diff
@@ -404,7 +404,7 @@ export default {
           qty: this.getCurrentProduct.qty
         })
 
-        this.manageQuantity = res.isManageStock
+        this.manageQuantity = Boolean(res.isManageStock)
         this.maxQuantity = res.isManageStock ? res.qty : null
```

Comment:
It would be nice to also replace `res.isManageStock` to `this.manageQuantity` here, then please update a changelog and we can merge it :)

Reply:
I've updated the Vue file https://github.com/DivanteLtd/vsf-default/pull/27/commits/66484769fa54712dd5c54471794e5fd4a956861d

There is no changelog on this project... So I've not updated it...

Reply:
@Fifciu can you merge my commit?

Reply:
I culd but please look, there is a changelog: https://github.com/DivanteLtd/vsf-default/blob/develop/changelog.md
Please update it

Reply:
Ok, done: https://github.com/DivanteLtd/vsf-default/pull/41

Reply:
Ok, thanks. It should be part of this PR. Please keep that in mind for the future @rozzilla . I am going to merge both this time to do not complain anymore, but please remember about it ;)

Reply:
Ok, sorry about that. Next time I'll do it correctly ;)

Reply:
I see you are trying to merge into master. You should merge into `develop` - please rebase this PR - and it would be the best if you merge these 2 in one.

Sorry for complaining so much but it is how it should be done.

Reply:
Unfortunately, I had to remove previous comment because I cannot merge it to `master`. Only to `develop`. Master is for releases

Reply:
Ok, so now I need to redo the PR on the develop branch?

Reply:
Yes, probably rebase your branch on `develop` and creating a new PR would be the easiest way. Same as @lukaszjedrasik did here with bugfix #25 https://github.com/DivanteLtd/vsf-default/pulls?q=is%3Apr+is%3Aclosed

Reply:
Done! https://github.com/DivanteLtd/vsf-default/pull/42

Reply:
Great, thanks!